docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

```diff
@@ -68,11 +68,11 @@ commands:
 +
 [NOTE]
 
-* A command can have only one action, though you can use `composite` commands to execute several commands either sequentially or in parallel.
+* A command only has one action. Still, you can use `composite` commands to execute several commands either sequentially or in parallel.
 
 . Command group
 +
-A given command can be assigned to one or more groups that represents the nature of the command.  The support groups are: `build`, `run`, `test` and `debug`. For each of the groups, one default command can be defined in each group by specifying the `isDefault` value.
+You can assign a given command to one or more groups that represents the nature of the command.  The support groups are: `build`, `run`, `test` and `debug`. For each of the groups, define one default command in each group by specifying the `isDefault` value.
```

Comment:
support groups are: `build`, `run`, `test` and `debug`.

Suggest: to bullet list them. Improves readability.